cfme/common/provider.py

```diff
@@ -167,6 +167,14 @@ def get_yaml_data(self):
             raise ProviderHasNoKey(
                 'Provider {} has no key, so cannot get yaml data'.format(self.name))
 
+    def has_nodes(self):
```

Comment:
Personally I feel this should be raised as a bug with manageiq devel, it's darn inconsistent